packages/wdio-utils/tests/envDetector.test.ts

```diff
@@ -11,25 +11,27 @@ import edgedriverResponse from './__fixtures__/edgedriver.response.json'
 import seleniumstandaloneResponse from './__fixtures__/standaloneserver.response.json'
 
 describe('sessionEnvironmentDetector', () => {
-    const chromeCaps = chromedriverResponse.value
-    const appiumCaps = appiumResponse.value.capabilities
-    const experitestAppiumCaps = experitestResponse.appium.capabilities
-    const geckoCaps = geckodriverResponse.value.capabilities
-    const edgeCaps = edgedriverResponse.value.capabilities
-    const phantomCaps = ghostdriverResponse.value
-    const safariCaps = safaridriverResponse.value.capabilities
-    const safariLegacyCaps = safaridriverLegacyResponse.value
-    const standaloneCaps = seleniumstandaloneResponse.value
+    const chromeCaps = chromedriverResponse.value as unknown as WebDriver.Capabilities
```

Comment:
double casting?

Reply:
removing the castings doesn't give me any errors, is this needed?

Reply:
it was the only chance for me to case timeout strings correctly, I will remove and see what the build says